src/server/lib/renderer.tsx

```diff
@@ -25,20 +25,20 @@ const favicon =
 
 interface RendererOpts {
   pageTitle: string;
-  serverState: ServerState;
+  requestState: RequestState;
 }
 
 const { gaUID } = getConfiguration();
 
 // function to map from req.locals, to the ClientState used by the client
-const clientStateFromServerStateLocals = (
+const clientStateFromRequestStateLocals = (
   {
     csrf,
     globalMessage,
     pageData,
     queryParams,
     abTesting,
-  } = getDefaultServerState(),
+  } = getDefaultRequestState(),
```

Comment:
is this default ever used?

Reply:
I'm not convinced it is. But because of the scope of changes here, I've made a note of it and I'll look at this when I do AB testing ticket.

Reply:
I think this is a leftover from our inital state management, where the state was optional